src/Compilers/Core/MSBuildTask/Shared/Microsoft.CSharp.Core.targets

```diff
@@ -95,7 +95,7 @@
           FileAlignment="$(FileAlignment)"
           GenerateFullPaths="$(GenerateFullPaths)"
           HighEntropyVA="$(HighEntropyVA)"
-          Instrument="$(Instrument)"
+          InstrumentationKinds="$(Instrument)"
```

Comment:
>Instrument [](start = 34, length = 10)

Should this be renamed too? Also, it looks like things that might have multiple values are using Items rather than a property.